test/controllers/staff_dashboard_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class StaffDashboardControllerTest < ActionController::TestCase
+=begin
```

Comment:
no i print request.original_url and it goes to /staff_dashboard when it shouldn't. check most recent commit